src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,40 +1,83 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+import { actions as currentTodoActions } from '../../features/currentTodo';
 
 export const TodoModal: React.FC = () => {
+  const dispatch = useAppDispatch();
+  const selectedTodo = useAppSelector(state => state.currentTodo);
+
+  const [user, setUser] = useState<User | null>();
+  const [isLoading, setIsLoading] = useState(false);
+
+  const removeTodo = () => dispatch(currentTodoActions.removeTodo());
+
+  const loadUser = async () => {
+    try {
+      setIsLoading(true);
+      if (selectedTodo) {
+        const userFromServer = await getUser(selectedTodo.userId);
+
+        setUser(userFromServer);
+      }
+    } catch {
+      throw new Error();
```

Comment:
throwing a new (+ empty) error is not correct error handling
since the error you throw here will not be handled anywhere - it will lead to an error and page crush. You should show the `Error` component, etc (in the same way you do in the App.tsx)